### ISSUE TITLE
Edit function for multiple selections in Timesheets

### ISSUE THREAD

**Issue by HeinzWuert** [kevinpapst/kimai2#2211]
**Is your feature request related to a problem? Please describe.**
If I mark several entries inTimesheets for moving them into another project and click on Edit at the bottom, the input window for time bookings opens.
There are three mandatory fields:
- Customer
- Project
- Activity

The Activity field is the problem. If I want to move several data records to another project, this is not possible in one operation if the activities are different. I'm forced to repeat this process for every existing activity group.

**Describe the solution you'd like**
It would be a lot easier if activities weren't required!

**Comment by kevinpapst** [comment]
How to handle the case that three entries had three different projects WITH project specific activities?

**Comment by HeinzWuert** [comment]
what sense does it make to edit three different projects with this function?
In my case I used the filter for selecting a specfic project to move the bookings in another one.

**Comment by kevinpapst** [comment]
None 😁 

But you could still open such entries and I have to handle that case: that's why the activity is currently a required field.

**Comment by HeinzWuert** [comment]
Indeed, you must really know what you are doing with this function.
But I don't see the sense of the mandatory fields or why there are three.
Choose projects which ever you want. 
Edit one or more fields and confirm.
Change for the selected bookings only these fields, where an entry was made and leave all other fields original.

**Comment by adamgundry** [comment]
I've also hit a use case for this: we're using global activities exclusively, and I wanted to move all time entries from one project to another, but doing so currently seems to be difficult. I guess ideally the batch update screen would detect when any of the entries being updated had project specific activities, and allow preserving them if they were all global (or perhaps, have a way to override only project-specific activities while leaving the global ones unchanged?). Thanks in advance for taking a look!

**Comment by KIG-SB** [comment]
Having this issue as well now. We need to move certain records from an initial development project to a customer specific project. The activities that are used are global. This might happen again in the future.

Another option as potential solution: add a "force" checkbox on the edit page, maybe showing it only after getting this error.

**Comment by SaturnusDJ** [comment]
(Woops, posted with an internal account initially.)

Having this issue as well now. We need to move certain records from an initial development project to a customer specific project. The activities that are used are global. This might happen again in the future.

Another option as potential solution: add a "force" checkbox on the edit page, maybe showing it only after getting this error.

**Comment by gskerry** [comment]
Same use case, and issue. Would be beneficial to have option to leave (global) activity settings unchanged when updating project. Thanks!
